qa/tasks/cephfs/test_failover.py

```diff
@@ -265,26 +265,23 @@ def test_hole(self):
 
         self.fs.grow(2)
 
+        # Now add a delay which should slow down how quickly rank 1 stops
+        self.config_set('mds', 'ms_inject_delay_max', '5.0')
+        self.config_set('mds', 'ms_inject_delay_probability', '1.0')
         self.fs.set_max_mds(1)
         log.info("status = {0}".format(self.fs.status()))
 
-        self.fs.set_max_mds(3)
         # Don't wait for rank 1 to stop
+        self.fs.set_max_mds(3)
```

Comment:
nit: could `grow/shrink` filesystem helpers be used here with some tweaks to not wait for daemons to be healthy?

Reply:
I kinda like how the `set_max_mds` is explicit about what it's doing. `grow`/`shrink` may do other things in the future which could disrupt this test.

Reply:
fair enough.